python/lsst/resources/davutils.py

```diff
@@ -324,7 +322,7 @@ class DavConfigPool:
 
     Parameters
     ----------
-    filenames : `list[str]`
+    filename : `list` [ `str` ]
```

Comment:
Not quite sure if it really matters, but I don't think you need spaces between the brackets and the item type. Only a space between the container type and the brackets wrapping the item type is required ([src](https://developer.lsst.io/python/numpydoc.html#sequence-types)).